### PR TITLE
InnerSource Portal Extensions (incl Banco Santander)

### PR DESCRIPTION
**Pull request by spier**
PR contains extensions to the InnerSource Portal pattern.

Mostly formatting changes but also some changes to the content, which make it easier to integrate the Banco Santander use case to the Known Instances of the pattern.

## Content Changes

- Adding Banco Santander to Known Instances
- Break out description of portal properties into a list, and add exploration about scenarios where publishing the portal publicly makes sense (rather than on the intranet)
- Rewording the description of the communication campaign.
- Rewriting Patlet into Problem+Solution
- Improve cropping of the portal image (and port the image over into this repo)

## Formatting Changes

- Changing order of Context and Forces
- Turning Context into bullet point format (to be used as a checklist by the readers) 
- Turning Resulting Context into bullet point format (to be used as a checklist by the readers)
- Making company names in Known Instances bold
- Fixing link to gitlab.com/gilda2

## Potential for further improvements, not part of this PR

- improve the Problem section
- move pattern to maturity "Validated"? (we have not done this before, so no prior art exists)

### CODE REVIEW

**Comment by dicortazar** [comment]
@spier this looks great, thanks a lot!

Side question: is there a style guide I can follow when doing reviews?

**Comment by spier** [comment]
@dicortazar thanks for the quick feedback.

Our [handbook](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/contributor-handbook.md#requirements-level-2---structured) describes some requirements for a pattern in a given level. Does that cover what you are looking for in a "style guide", or do you mean something else?

File: patterns/2-structured/innersource-portal.md [hunk 1]
```diff
@@ -69,3 +84,10 @@ The InnerSource Portal has enabled InnerSource project owners to advertise their
 ## Author(s)
 
 * Stephen McCall
+
+## Acknowledgements
+
+* Shelly Nizri
+* Melinda Malmgren
+* Michael Graf
+* Jesus Alonso
```
Comment (jesusalonsogut) [inline patterns/2-structured/innersource-portal.md]: Jesus Alonso Gutierrez

Comment (spier) [inline patterns/2-structured/innersource-portal.md]:
```suggestion
* Jesús Alonso Gutierrez
```

**Review by dicortazar — APPROVED** [review]
This looks great, thanks for the work!

**Pull request merged**